mypy.ini

```diff
@@ -6,11 +6,9 @@ show_column_numbers = True
 check_untyped_defs = True
 pretty = True
 
-files =
-    torchdata,
-    test
+files = torchdata
 
-exclude = examples
+exclude = examples, test
```

Comment:
That means `mypy` won't scan through any test files right?

Reply:
Yeah. It's way too hard to maintain `mypy` for tests if we want to do some hacks.
Actually, that part is redundant as I only set `files = torchdata`. None of other folds will carry out mypy.

Reply:
Second thought, let me keep the `exclude` here to make everything explicitly.